docs/basics/faq-troubleshooting.md

```diff
@@ -245,19 +261,25 @@ Using [`risky`](./options.md#matchmode) matching will match torrents for snatchi
 Using [`safe`](./options.md#matchmode) matching will match torrents for snatching based on the size and require that the release groups match and compare them, requiring file structure to match.
 :::
 
-:::caution
+:::danger WARNING
 There are TV shows that are not combined into season packs, such as Jeopardy, Talk Shows, or similar. These shows often have naming which specifies dates, rather than SeasonEpisode style. Searching with `includeSingleEpisodes` and using `risky` on these torrents, or files, can result in _many torrent files being downloaded_ for comparisons for matching only _one individual episode_.
 
 Remember, this would occur on just one (torrent) search, within the entire (all data) search.
 :::
 
 ---
 
-It is highly recommended to use [`excludeOlder` and `excludeRecentSearch`](./daemon.md#set-up-periodic-searches). Both of these are covered [here](./daemon.md#set-up-periodic-searches) as well as on the [options page](./options.md#excludeolder).
+##### Final Notes
+
+:::tip Be ADVISED
+**There is no one-size-fits-all setting for this**, but you can run `cross-seed` with your config and see how many torrents it will search (noted as "suitable" prior to conducting the actual search) and adjust accordingly.
```

Comment:
```suggestion
**There is no one-size-fits-all setting for this**. You can run `cross-seed` with your config to see how many torrents it will search (noted as "suitable" before conducting the actual search) and adjust accordingly.
```
Remove "but". People ignore everything before the "but"